docs/b9lab-processes.md

```diff
@@ -0,0 +1,93 @@
+# B9lab processes
+
+This file describes the internal processes for the [B9lab](https://b9lab.com) contribution team. External contributors should follow the [Contribution Guide](../CONTRIBUTING.md).
+
+
+## General
+
+**We are exclusively working on the public github repository at [https://github.com/cosmos/sdk-tutorials](https://github.com/cosmos/sdk-tutorials)!**
+
+Our default branch naming scheme applies: A branch should be named `xx-branch-description`, there xx should be replaced with your initials (lowercase), and the branch description should give enough context to understand what is in this branch. For example, this file has been updated through a branch named `is-update-guidelines`.
+
+
+## Creating a new Pull Request
+
+When creating a new Pull Request, you **must** follow these steps:
+
+1. Select the `b9lab_pr_template` as Pull request template.
+2. Set a [descriptive name](https://cbea.ms/git-commit/), including a category prefix if applicable:
+   * `docs:` for changes on the content,
+   * `fix:` for updates fixing issues on the platform. These do not include fixes on the text content itself, which fall in the `docs:` category,
+   * `feature:` for updates on the platform features/functionality.
+3. Set the Pull Request description, filling all required fields.
+4. Apply the **label** `b9lab-internal`.
+5. Set up Assignee if required: Assignees should be people who need to work on something. Also make sure all assignees know what they need to do, using the description and/or comments.
+6. Set up the Pull Request in Draft initially (click on the arrow next to the _Create Pull Request_ button and select _Create Draft Pull Request_).
+
+
+The Pull Request is now ready to be moved into the internal review state.
+
+
+## R1 Start internal review
+
+Once your Pull Request is ready for internal review (content and/or language), you can initiate the internal review process:
+
+1. Check the checkboxes on the Pull Request description accordingly to the review you want to request, in the internal review section:
+
+- Technical content review
```

Comment:
In case we change it in file `/b9lab_pull_request_template.md`, we would also have to include the wording change here and in the following steps.